### DISCUSSION TITLE
TypeError: Relationship() got an unexpected keyword argument 'cascade_delete'

### DISCUSSION THREAD

**Discussion by oliviedu95** [fastapi/full-stack-fastapi-template#1310] [Questions] [answered]
### First Check

- [X] I added a very descriptive title here.
- [X] I used the GitHub search to find a similar question and didn't find it.
- [X] I searched in the documentation/README.
- [X] I already searched in Google "How to do X" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs/README and didn't find an answer.

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```Text
I just try to start the 0.7.0 version without editing any code and get this error:
backend-1  |   File "/app/app/models.py", line 46, in User
backend-1  |     items: list["Item"] = Relationship(back_populates="owner", cascade_delete=True)
backend-1  | TypeError: Relationship() got an unexpected keyword argument 'cascade_delete'
```

### Description

* docker compose up -d
* docker compose logs backend

### Operating System

Other

### Operating System Details

WSL

### Python Version

Python 3.10.6

### Additional Context

TypeError: Relationship() got an unexpected keyword argument 'cascade_delete'

**Comment by alejsdev** [comment, marked as answer]
Hi @oliviedu95 

The error you're encountering is probably due to a recent update in SQLModel. You need to rebuild your Docker containers to ensure all dependencies are updated properly. You can run `docker compose build` before `docker compose up -d`.

Hope it works!
